malsim/scenario.py

```diff
@@ -32,13 +33,13 @@
     'lang_file',
     'model_file',
     'attacker_agent_class',
-    'defender_agent_class',
```

Comment:
We've discussed this previously. I would always have both types of agents be required and you can just set them to `None`/`False` if you don't want one.

This is because I don't want people to accidentally not provide certain agents in their scenarios. They should explicitly state that that scenario does not contain an Attacker or Defender.

Reply:
I think it makes sense, will revert it.